src/OrchardCore/OrchardCore.Setup.Core/SetupService.cs

```diff
@@ -181,7 +181,7 @@ private async Task<string> SetupInternalAsync(SetupContext context)
                 case DbConnectionValidatorResult.InvalidConnection:
                     context.Errors.Add(String.Empty, S["The provided connection string is invalid or server is unreachable."]);
                     break;
-                case DbConnectionValidatorResult.DocumentFound:
+                case DbConnectionValidatorResult.ShellDescriptorDocumentFound:
```

Comment:
Is this by intention? I see `DocumentFound` replaced by `DocumentTableFound`

Reply:
yes